src/plugins/plugin.datavalue.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	// var noop = helpers.noop;
+	Chart.defaults.global.valueLabel = {
+		// position: 'top',
+		fontWeight: 'normal',
+		fontColor: '#000',
+		fontSize: 10,
+		padding: {top: 0, left: 0}
+		// truncate: true
+	};
+	function DrawValue(chart, element, value, index, chartype) {
```

Comment:
I think this should be `drawValue` since it's not a constructor. Please also add JSdoc comments.